ModList/UI/BSML/MarkdownTextHandler.cs

```diff
@@ -18,21 +15,46 @@ public class MarkdownTextHandler : TypeHandler<MarkdownText>
     {
         public override Dictionary<string, string[]> Props { get; } = new Dictionary<string, string[]>
         {
-            { "text", new[] { "text", "value" } },
-            { "childText", new[] { "_children" } },
-            { "linkPressed", new[] { "link-pressed" } },
-            { "linkColor", new[] { "link-color" } },
-            { "autolinkColor", new[] { "autolink-color" } },
+            {
+                "text", new[]
+                {
+                    "text",
+                    "value"
+                }
+            },
+            {
+                "childText", new[]
+                {
+                    "_children"
+                }
+            },
+            {
+                "linkPressed", new[]
+                {
+                    "link-pressed"
+                }
+            },
+            {
+                "linkColor", new[]
+                {
+                    "link-color"
+                }
+            },
+            {
+                "autolinkColor", new[]
+                {
+                    "autolink-color"
+                }
+            }
         };
 
-        public override Dictionary<string, Action<MarkdownText, string>> Setters { get; } 
-            = new Dictionary<string, Action<MarkdownText, string>>
-            {
-                { "text", (md, text) => md.Text = text },
-                { "childText", HandleTextAsChildren },
-                { "linkColor", (md, col) => md.LinkColor = GetColor(col, md.LinkColor) },
-                { "autolinkColor", (md, col) => md.AutolinkColor = GetColor(col, md.AutolinkColor) },
-            };
+        public override Dictionary<string, Action<MarkdownText, string>> Setters { get; } = new Dictionary<string, Action<MarkdownText, string>>
```

Comment:
This can use C# 9 and just be `new() ...`

Reply:
C# 9 is sadly not allowed for BeatMods mods for the time being as per request of @Auros

Reply:
That seems... odd. Did he give a reason?

Reply:
I requested for mods to not use C# 9 since dnSpy can't fully decompile some C# 9 code without it looking funky, and other popular deobfuscators that are used by approvers like JustAssembly will just straight up crash.

If I were the only BeatMods approver i probably wouldn't have an issue and go the extra mile to verify stuff, but I'm not.

Reply:
IIRC the only things C# 9 thing that dnSpy *really* doesn't like is function pointers. Most stuff works fine.